pages/download.html

```diff
@@ -212,8 +212,11 @@ <h1>{% blocktrans %}Download Now ({{ stable_version }} Stable){% endblocktrans %
   <div class="fullwidth">
     <p>{% blocktrans %}Check out the <a href="{{ stable_release_announcement }}">release announcement</a> for a list of new features.{% endblocktrans %}</p>
     <p>{% blocktrans with previous_link="#previous" %}Although {{ stable_version }} contains many important updates and bugfixes, it's possible some users may experience issues. If you do, you can still get our previous stable version, <a href="{{ previous_link }}">{{ previous_version }}</a>.{% endblocktrans %}</p>
-    <p>{% trans "Mixxx is available for Windows, macOS, and Linux." %}<br/>
-    {% trans "We happily provide Mixxx for free and donations are appreciated!" %}</p>
+    <p>{% trans "Mixxx is available for Windows, macOS, and Linux." %}</p>
+    <h3>{% trans "You don't need to pay for Mixxx because you can get it here for free." %}<br/>
+        {% trans "There is no official paid or &quot;pro&quot; version of Mixxx." %}<br/>
+        {% trans "The version available here is the full version." %}</h3>
```

Comment:
remove the `<h3>`

Reply:
Yup. Even if we decide to keep the emphasis we should remove the `h3` tag as it denotes a headline, not emphasized text. HTML should be used semantically, so that users that depend on a screen reader can also read it properly.

Reply:
ok but the text was too small just as paragraph text.  what tag do you want me to use?

Reply:
I think it should be paragraph text like the rest of the text surrounding it.

Reply:
I don't think it's prominent enough, but ok